Assets/VIRTUOSO/VRTK/Source/SDK/Unity/SDK_UnityController.cs

```diff
@@ -157,7 +315,12 @@ public override void ProcessFixedUpdate(VRTK_ControllerReference controllerRefer
         public override ControllerType GetCurrentControllerType(VRTK_ControllerReference controllerReference = null)
         {
             SetTrackedControllerCaches();
+#if !UNITY_2019_1_OR_NEWER
             return cachedControllerType;
+#else
+            // not used post-2019
```

Comment:
pre-2019

Reply:
rearranged this as the define setup was a little confusing in this spot